.gitignore

```diff
@@ -1,5 +1,3 @@
 coverage
 node_modules
-npm-debug.log
-lib
-dist
\ No newline at end of file
+npm-debug.log
```

Comment:
These files are left out of the library by design to avoid the maintenance concerns they bring. Glad to take a look at a PR with just functional changes.